pygmt/src/ternary.py

```diff
@@ -0,0 +1,66 @@
+"""
+ternary - Plot data on ternary diagrams.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    F="center",
+    G="fill",
+    JX="width",
+    R="region",
+    S="style",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", T="sequence", c="sequence_comma", p="sequence")
+def ternary(self, table, **kwargs):
```

Comment:
We should use `data`, not `table`, following the discussions in https://github.com/GenericMappingTools/pygmt/issues/1479.

Reply:
done in ce6de7c